tests/test_dependabot.py

```
@@ -0,0 +1,454 @@
+                repo="o/r",
+                github_user="u",
+                config=DependabotConfig(enabled=True),
+            )
+        assert len(results) == 1
+        assert results[0].pr_number == 2
+
+    @pytest.mark.asyncio
+    async def test_sweep_skips_major_bumps(self) -> None:
+        prs = [self._make_gh_pr(1, title="bump django from 5.2 to 6.0")]
+
+        async def _mock_process(pr, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_mock_process`

(ANN202)